public/client/TracyProfiler.cpp

```diff
@@ -1480,26 +1526,6 @@ void Profiler::SpawnWorkerThreads()
     m_exceptionHandler = AddVectoredExceptionHandler( 1, CrashFilter );
```

Comment:
You missed this.

Reply:
oops, sorry just testing on linux, hopefully next commit fixes.